--- structs.go.old	2021-10-01 14:56:44.944208960 +0000
+++ structs.go	2021-10-01 14:56:41.660150490 +0000
@@ -310,12 +310,12 @@
 	Name                 string                 `json:"name,omitempty"`
 	Topic                string                 `json:"topic,omitempty"`
 	NSFW                 bool                   `json:"nsfw,omitempty"`
-	Position             int                    `json:"position"`
+	Position             int                    `json:"position,omitempty"`
 	Bitrate              int                    `json:"bitrate,omitempty"`
 	UserLimit            int                    `json:"user_limit,omitempty"`
 	PermissionOverwrites []*PermissionOverwrite `json:"permission_overwrites,omitempty"`
 	ParentID             string                 `json:"parent_id,omitempty"`
-	RateLimitPerUser     int                    `json:"rate_limit_per_user,omitempty"`
+	RateLimitPerUser     *int                   `json:"rate_limit_per_user,omitempty"`
 }
 
 // A ChannelFollow holds data returned after following a news channel
